org.freedesktop.LinuxAudio.Plugins.Cardinal.yml

```diff
@@ -0,0 +1,52 @@
+id: org.freedesktop.LinuxAudio.Plugins.Cardinal
+runtime: org.freedesktop.LinuxAudio.BaseExtension
+sdk: org.freedesktop.Sdk//21.08
+runtime-version: '21.08'
+build-extension: true
+appstream-compose: false
+branch: '21.08'
+build-options:
+  prefix: /app/extensions/Plugins/Cardinal
+  prepend-pkg-config-path: /app/extensions/Plugins/Cardinal/lib/pkgconfig
+  prepend-ld-library-path: /app/extensions/Plugins/Cardinal/lib
+modules:
+  - name: jansson
+    buildsystem: cmake-ninja
+    sources:
+      - type: archive
+        url: https://github.com/akheron/jansson/releases/download/v2.14/jansson-2.14.tar.bz2
+        sha256: fba956f27c6ae56ce6dfd52fbf9d20254aad42821f74fa52f83957625294afb9
+  - name: Cardinal
+    buildsystem: simple
+    build-commands:
+      - 'make PREFIX=${FLATPAK_DEST} SYSDEPS=true -j $FLATPAK_BUILDER_N_JOBS'
+      - 'make PREFIX=${FLATPAK_DEST} install'
```

Comment:
this should use PREFIX and DESTDIR, not PREFIX combined. otherwise the wrong path will be set as prefix in the source.